app/views/coronavirus_form/check_your_answers/helpers/_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if items.any? %>
+  <ul class="govuk-list govuk-list--bullet">
+    <% items.each do |item| %>
+      <li><%= item %></li>
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
There are a few files like this that done have a blank line at the end. That can cause problems sometimes.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Reply:
Thanks @reggieb 👍  Good spot - we will fix this.